app/services/abstract_manual_document_service_registry.rb

```diff
@@ -80,4 +88,46 @@ def manual_document_builder
   def manual_repository
     raise NotImplementedError
   end
+
+  def organisation(slug)
+    @organisations ||= {}
+    @organisations[slug] ||= organisations_api.organisation(slug)
+    @organisations[slug]
+  end
+
+  def publishing_api_draft_manual_exporter
+    ->(_, manual) {
+      ManualPublishingAPIExporter.new(
+        publishing_api.method(:put_draft_content_item),
+        organisation(manual.attributes.fetch(:organisation_slug)),
+        SpecialistPublisherWiring.get(:manual_renderer),
+        PublicationLog,
+        manual
+      ).call
+    }
+  end
+
+  def publishing_api_draft_manual_document_exporter
+    ->(manual_document, manual) {
+      ManualSectionPublishingAPIExporter.new(
+        publishing_api.method(:put_draft_content_item),
+        organisation(manual.attributes.fetch(:organisation_slug)),
+        SpecialistPublisherWiring.get(:manual_document_renderer),
+        manual,
+        manual_document
+      ).call
+    }
+  end
+
+  def manual_document_renderer
+    SpecialistPublisherWiring.get(:manual_document_renderer)
+  end
+
+  def publishing_api
+    SpecialistPublisherWiring.get(:publishing_api)
+  end
+
+  def organisations_api
```

Comment:
Since this is now referenced from more than one class, it should probably go into the SpecialistPublisherWiring, as we did with publishing_api.